src/org/traccar/api/resource/PermissionsResource.java

```diff
@@ -28,30 +29,25 @@
 
 import org.traccar.Context;
 import org.traccar.api.BaseResource;
-import org.traccar.model.GroupDriver;
 
-@Path("groups/drivers")
+@Path("permissions")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
-public class GroupDriverResource extends BaseResource {
+public class PermissionsResource  extends BaseResource {
 
     @POST
-    public Response add(GroupDriver entity) throws SQLException {
+    public Response add(LinkedHashMap<String, Long> entity) throws SQLException, ClassNotFoundException {
```

Comment:
Idea for improvement. Have some helper class that will have methods to get owner and property keys and values. It can extend `LinkedHashMap`, so it might even work automatically as an argument here. I believe this will simplify some places where we "parse" this entity.